andela/travel_tool_front
# #162540377 Fix the delete modal bug

#### What does this PR do?
Fix bug on delete modal

#### Description of Task to be completed?
* Set close modal to true on successful request delete
* Update open request count when delete is successful

#### How should this be manually tested?
- Clone the repo
    `git clone https://github.com/andela/travel_tool_front.git`
 - Switch to the project directory
   `cd travel_tool_front`
 - Checkout the **bug-delete-modal-bug-162540377** branch
   `git checkout bug-delete-modal-bug-162540377`
 - Install dependencies and start the server
   - `make start` *if you have docker*
   - `yarn install` then `yarn start` *if you do not have docker*
 - Start the [backend](https://github.com/andela/travel_tool_back) of this project.
 - Navigate to [travela](http://travela-local.andela.com:3000/) in your browser and log in with your Andela account. 
- Navigate to [travela requests](http://travela-local.andela.com:3000/requests) in your browser.
 - Create at least 2 travel request
 - Delete one of the travel requests you just created.
 - You should see the update on the open request count.
 - Click on the ellipsis of the other request 
 - You should see the options without the delete modal popping up


#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
* [#162540377](https://www.pivotaltracker.com/story/show/162540377)

#### Screenshots (if appropriate)
<img width="1203" alt="screen shot 2018-12-11 at 6 06 05 pm" src="https://user-images.githubusercontent.com/20061324/49817021-78855000-fd6f-11e8-846d-740067212b59.png">

#### Questions:
N/A
